pkg/agent/qrm-plugins/network/staticpolicy/policy.go

```diff
@@ -769,6 +782,10 @@ func (p *StaticPolicy) applyNetClass() {
 			}(string(pod.UID), container.Name, netClsData)
 		}
 	}
+
+	if p.CgroupV2Env {
```

Comment:
Do we need to clear residual net class data for cgroup v1?

Reply:
In cgroup v1, it is not necessary to clear residual net class because the net class is set within the cgroup path. This will be automatically cleared when the pod is terminated by kubelet.